modules/Sources/Features/Home/HomeView.swift

```diff
@@ -21,12 +21,18 @@ public struct HomeView: View {
         WithViewStore(store) { viewStore in
             VStack {
                 balance(viewStore)
-
                 Spacer()
-
-                sendButton(viewStore)
-
-                receiveButton(viewStore)
+                VStack {
+                    if viewStore.isSendButtonDisabled {
+                        dismissButton(viewStore)
+                    }else {
+                        sendButton(viewStore)
+                    }
```

Comment:
Because there is a modifier `disableButtonStyle`, it would be much better to add a parameter to it when it's disabled. So the use will become
```swift
sendButton(viewStore)
     .disableButton(when: viewStore.isSendButtonDisabled)
```

note: I removed suffix Style, than should be omit